src/js/themes/base.js

```diff
@@ -1682,6 +1682,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       // scrubber: { track: { color: undefined } },
       scrubber: {
         color: 'light-4',
+        interval: 10,
```

Comment:
Commented out Line 1682 looks suspect. Can we address the `scrubber.track.color` properties? Consolidate under the active scrubber property + get 100% alignment here and  `base.d.js`.